src/store/login/index.js

```diff
@@ -86,8 +91,7 @@ export const getAuthError = state => state.login.authError;
 export const getJwtToken = state => state.login.jwtToken;
 export const getIsAuthenticated = createSelector(getIsAuthSuccessful,
   getJwtToken,
-  (isAuthSuccessful,
-    jwtToken) => jwtToken && isAuthSuccessful === true);
+  (isAuthSuccessful, jwtToken) => (jwtToken && isAuthSuccessful) === true);
```

Comment:
The final equal is not necessary at all.

Reply:
I'll create an additional selector, to help decide if the JWT token exists